reference/7/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -26,6 +27,7 @@ Integer literals can have a type suffix and a multiplier suffix.
 | l      | long data type                 |                         |
 | u      | unsigned int or long data type | Added in PowerShell 6.2 |
 | ul     | unsigned long data type        | Added in PowerShell 6.2 |
+| n      | bigint data type               | Added in PowerShell 7.0 |
```

Comment:
Given that `BigInteger` is technically a non-primitive type (it's a `byte`-based `struct`, I believe), would it be more appropriate to give its full name here? `System.Numerics.BigInteger`

Reply:
Updated.